docs/feature_tri_layer.md

```diff
@@ -0,0 +1,48 @@
+# Tri Layers :id=tri-layers
+
+This enables support for the OLKB style "Tri Layer" keycodes.  These function similar to the `MO` (momentary) function key, but if both the "Lower" and "Upper" keys are pressed, it activates a third "Adjust" layer.  To enable this functionality, add this line to your `rules.mk`:
+
+```make
+TRI_LAYER_ENABLE = yes
+```
+
+Note that the "upper", "lower" and "adjust" names don't have a particular significance.  They are just used to identify and clarify the behavior, and the layers don't have to be in any particular order.  However, layers are processed from highest numeric value to lowest.
```

Comment:
The 2 last sentences kind of contradict each other, something more like
```suggestion
Note that the "upper", "lower" and "adjust" names don't have a particular significance, they are just used to identify and clarify the behavior. Layers are processed from highest numeric value to lowest, however the values are not required to be consecutive.
```

Reply:
Yeah, that's a lot less ambiguous. Thanks